integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -63,4 +69,59 @@ public void remove_key(String key) throws IOException {
         map.remove(key);
         sessionRequestBody = objectMapper.writeValueAsString(map);
     }
+
+    @When("user sends a valid request to authorization end point")
+    public void user_sends_a_valid_request_to_authorization_end_point()
+            throws IOException, InterruptedException, URISyntaxException {
+        response =
+                IpvCoreStubUtil.sendAuthorizationRequest(
+                        DEV_AUTHORIZATION_URI,
+                        currentSessionId,
+                        "https://di-ipv-core-stub.london.cloudapps.digital/callback",
+                        "ipv-core-stub");
```

Comment:
declare: `private static final String DEFAULT_CLIENT_ID = ...`
for the client id and reference it where necessary